vendor/github.com/moby/buildkit/vendor.conf

```diff
@@ -23,7 +23,7 @@ github.com/Microsoft/go-winio v0.4.7
 github.com/containerd/fifo 3d5202aec260678c48179c56f40e6f38a095738c
 github.com/opencontainers/runtime-spec v1.0.1
 github.com/containerd/go-runc f271fa2021de855d4d918dbef83c5fe19db1bdd5
-github.com/containerd/console 9290d21dc56074581f619579c43d970b4514bc08
+github.com/containerd/console 5d1b48d6114b8c9666f0c8b916f871af97b0a761
```

Comment:
docker/cli is behind; https://github.com/containerd/console/compare/cb7008ab3d8359b78c5f464cb7cf160107ad5925...5d1b48d6114b8c9666f0c8b916f871af97b0a761

Reply:
updated with 0674a4d